### DISCUSSION TITLE
chdman for folders

### DISCUSSION THREAD

**Discussion by Papermanzero** [mamedev/discussions#130] [Ideas]
Currently chd supports discs and hdd images to make a lossless archive file. With the support of zstd aka zarchive mame introduced an even better way to compress image files. 
however due to digitalisation more and more images are file based (dlc, app games, updates…).

Is it possible to add the functionality in the next chdmame to compress folders with several files?
This would life to archive files much more easier.

**Comment by einstein95** [comment]
That's why MAME uses zip/7z files.

**Reply by Papermanzero** [reply]
But cannot we combine it? Just asking. It would be comfy to have one archive file type. Even with 7zip the reduction of size with chd is much more efficient.

**Reply by cuavas** [reply]
MAME supports zip archives with Zstd compression.

**Comment by rb6502** [comment]
CHDMAN and the CHD file format are for media images (magnetic and optical disks and tapes).  There are many other solutions for collections of files, including zip, 7zip, and tar + gzip, bzip, or Zstd.
